isthmus/src/test/java/io/substrait/isthmus/PlanTestBase.java

```diff
@@ -23,10 +24,20 @@
 import org.junit.jupiter.api.Assertions;
 
 public class PlanTestBase {
+  final SimpleExtension.ExtensionCollection extensions;
+
+  {
+    try {
+      extensions = SimpleExtension.loadDefaults();
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
   protected final RelCreator creator = new RelCreator();
   protected final RelBuilder builder = creator.createRelBuilder();
   protected final RexBuilder rex = creator.rex();
-  protected final RelDataTypeFactory type = creator.type();
+  protected final RelDataTypeFactory typeFactory = creator.typeFactory();
```

Comment:
I'm using this in my new tests, but it wasn't used before. Updated the named to `typeFactory` because I found `type` a confusing name for this.